src/state/pools/helpers.ts

```diff
@@ -43,6 +52,7 @@ export const transformPool = (pool: SerializedPool): DeserializedPool => {
     userData: transformUserData(userData),
     totalStaked: new BigNumber(totalStaked),
     stakingLimit: new BigNumber(stakingLimit),
+    numberBlocksForUserLimit: new BigNumber(numberBlocksForUserLimit),
```

Comment:
could we centralize `Max. stake limit ends`  calculation in here ?

Reply:
Done